backend/src/Squidex.Domain.Apps.Entities/Contents/Queries/ContentQueryService.cs

```diff
@@ -93,11 +93,9 @@ public async Task<IResultList<IEnrichedContentEntity>> QueryAsync(Context contex
 
             var schema = await GetSchemaOrThrowAsync(context, schemaIdOrName);
             var permissionReadOwn = Permissions.ForApp(Permissions.AppContentReadOwn, context.App.Name, schemaIdOrName);
```

Comment:
Checking for permissionReadOwn  here is not necessary (See next comment).

Reply:
again.... If user doesn't have contents.read but has only contents.read.own  need I change to: 
```
if (context.Permissions.Allows(permissionRead))
            {
                q.CreatedBy = context.User.Token();
            }
```

Reply:
Perhaps you should write some tests